### PR TITLE
[14.0][MIG] base_action_visibility_restriction

### PR DESCRIPTION
**Pull request by dessanhemrayev**
Migration to 14.0

### CODE REVIEW

**Comment by francesco-ooops** [comment]
Hi @JordiMForgeFlow 

I'm trying to follow readme USAGE:

Go to Settings > Technical > Actions.
Search for any server/window action and edit it.
Update "Excluded groups" with one group.

but actually I can't see "Excluded groups" in v13 as well, am I doing something wrong?

image

File: base_action_visibility_restriction/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+To use this module, you need to:
+
+#. Activate the developer mode
+#. Go to *Settings > Technical > Actions*.
```
Comment (francesco-ooops) [inline base_action_visibility_restriction/readme/USAGE.rst]: #. Go to *Settings > Technical* and select either *Windows Actions* or *Server Actions*.

**Comment by JordiMForgeFlow** [comment]
Hi @francesco-ooops 

It is from the specific server/window actions menus (Settings > Techincal > Actions > Window Actions and Settings > Techincal > Actions > Server Actions). I think we could update the README to avoid confusion if it looks fine for you :)

File: base_action_visibility_restriction/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+#. Search for any server/window action and edit it.
+#. Update "Excluded groups" with one group.
```
Comment (francesco-ooops) [inline base_action_visibility_restriction/readme/USAGE.rst]: #. Open tab *Security* and update "Excluded groups" with one group.

**Review by francesco-ooops — COMMENTED** [review]
@dessanhemrayev can you add this in new commit?

**Comment by francesco-ooops** [comment]
@JordiMForgeFlow yes thank I figured out! We can add the change here and then backport it

**Review by francesco-ooops — APPROVED** [review]
Functional ok

**Comment by francesco-ooops** [comment]
@dessanhemrayev please add meaningful commit name

**Comment by francesco-ooops** [comment]
@JordiMForgeFlow good for you? :)

**Review by JordiMForgeFlow — APPROVED** [review]
LGTM 👍🏼 Thank you! :)

**Comment by francesco-ooops** [comment]
@OCA/tools-maintainers shall we? :)

**Comment by thomaspaulb** [comment]
We shall,

/ocabot merge nobump

/ocabot migration base_action_visibility_restriction

**Pull request merged**